cmd/revad/svcs/grpcsvcs/storageprovidersvc/storageprovidersvc.go

```diff
@@ -410,7 +410,7 @@ func (s *service) ListContainer(ctx context.Context, req *storageproviderv0alpha
 		return res, nil
 	}
 
-	var infos = make([]*storageproviderv0alphapb.ResourceInfo, 0, len(mds))
+	infos := []*storageproviderv0alphapb.ResourceInfo{}
```

Comment:
The reason to use make is to reduce dynamic expansion of the slice, we know beforehand the size it can growth, so it is cheaper to pre-allocate the space, can you revert back?

Reply:
yep, I got that after seeing some other lines ;-)